GDevelop.js/__tests__/Core.js

```diff
@@ -3676,6 +3676,31 @@ describe('libGD.js', function () {
         false
       );
     });
+
+    it('should rename a scene object in project groups', function () {
```

Comment:
Should a scene object be allowed in a global group in the 1st place?
What do other scenes show in the group as they don't know the object?

Reply:
> Should a scene object be allowed in a global group in the 1st place
  
Probably not. Like in a programming language, you can refer to things higher in the scope, but not lower in the scope.
This is supported for now because groups are based on object names... but that's an edge case we can continue to partially support (i.e: it works, but refactoring does not) and warn against.

> What do other scenes show in the group as they don't know the object?

They won't show anything (showing it's problematic).

But I think this is even worse because if you have this:
- A Global group "Enemies" containing two names: "EnemyBlue", "EnemyRed"
- A scene "Forest" with two objects: "EnemyBlue", "EnemyRed"
- A scene "Lava" with two objects: "EnemyBlue", "EnemyRed"

All is good! Then if you rename "EnemyBlue" in "Forest", this change will now automatically rename the global group (nice!) but not the object in the scene "Lava". Oops, you now have a broken scene without touching it!